frontend/src/util/encoding.ts

```diff
@@ -33,3 +34,31 @@ export const decodeData = async (data: string) => {
   const results = await getGUnzip
   return decode(results) as StoredProject
 }
+
+export const encodeModule = async (data: StoredModule) => {
```

Comment:
Probably can be rolled into the existing function. Type should probably be `any` since its a generic `encodeData` function